.github/workflows/renovate.json

```diff
@@ -0,0 +1,34 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "dependencyDashboard": true,
+    "onboarding": false,
+    "automerge": false,
+    "separateMinorPatch": true,
+    "platform": "github",
+    "extends": ["config:base"],
+    "labels": ["version_bump"],
+    "forkProcessing": "enabled",
+    "semanticCommits": "enabled",
+    "prConcurrentLimit": 20,
+    "prHourlyLimit": 20,
+    "dockerfile": {
+        "enabled": false
+    },
+    "customManagers": [
+        {
+        "customType": "regex",
+        "fileMatch": ["embedded-bins/Makefile.variables"],
+        "matchStrings": ["alpine_version\\s*=\\s*(?<currentValue>\\S+)"],
```

Comment:
Just remembering that we distinguish between the full Alpine version (x.y.z) and only the major/minor part, so I think we need to augment this?